### ISSUE TITLE
alive ?

### ISSUE THREAD

**Issue by KuKuRuZ1337** [openzfsonwindows/ZFSin#345]
Is this project alive or abandoned? 
When will zstd support be available ?

**Comment by lundman** [comment]
Just busy pushing out a macOS release, and updating the PRs. Then back here to update to latest, which will probably include zstd and the fresh raidz expansion.

**Comment by lundman** [comment]
Oh also, the main repository is now https://github.com/openzfsonwindows/openzfs

**Comment by KuKuRuZ1337** [comment]
> Just busy pushing out a macOS release, and updating the PRs. Then back here to update to latest, which will probably include zstd and the fresh raidz expansion.

![изображение](https://user-images.githubusercontent.com/65289168/153663929-dacceb43-96d8-4b95-8641-17e073c6199e.png)
Thank you for not abandoning the project.
I hope I can mount the NVME partition with zfs (I have archlinux with zstd there)

**Comment by KuKuRuZ1337** [comment]
> Oh also, the main repository is now https://github.com/openzfsonwindows/openzfs

Will the binary drivers for windows be released there now ?

**Comment by lundman** [comment]
Yes, the big releases are on GH but current newest is nightly https://openzfsonosx.org/wiki/Windows_builds

**Comment by quarkpower** [comment]
I want to thank you for this proyect. Would love to see it fully implemented.

**Comment by i2** [comment]
I would also like to thank you for this project. Would it be possible to join forces with `ZFS on Linux` / OpenZFS project so there is minimal feature disparity?

https://github.com/openzfsonwindows/openzfs When I see many commits ahead and many commit behind, I shiver! :)

**Comment by lundman** [comment]
https://github.com/openzfs/zfs/pull/12110
https://github.com/openzfsonwindows/openzfs

It's just a mammoth effort to make it happen, and does not just rely on me, but many of us need to carry that in.

If you also check the number of commits from ZOL in the last 4 days you'll see we are going to be at 1:1 in about 3 more days, give or take motivation.

**Comment by lundman** [comment]
Here's a build up to date with ZOL today. It needs testing to see if there is any regression
https://www.lundman.net/OpenZFSOnWindows-debug-2.1.99-1624-g27e584dab7-dirty.exe
